src/components/PopupWithAvatar.js

```diff
@@ -0,0 +1,28 @@
+import Popup from './Popup.js';
+
+export default class PopupWithAvatar extends Popup {
```

Comment:
You should use `PopupWithForm` class for the avatar form, there is no need to create a new form class

Reply:
True! I've deleted the PopupWithAvatar class and used the PopupWithForm class for the avatar form. Thank you.